### PR TITLE
Api updates for 2.0

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

**Comment by mikewalch** [comment]
Review b5543bed0d0e6b79664f8b8ece1d74b89cc8ae86 as the other commit are just formatting changes

File: src/main/java/org/apache/accumulo/testing/continuous/UndefinedAnalyzer.java [hunk 1]
```diff
@@ -174,21 +163,16 @@ public boolean contains(String row) {
...
 
       SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss,SSS");
 
       if (masterLogs != null) {
         for (File masterLog : masterLogs) {
 
-          BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream(masterLog), UTF_8));
           String line;
-          try {
+          try (BufferedReader reader = new BufferedReader(new InputStreamReader(
```
Comment (keith-turner) [inline src/main/java/org/apache/accumulo/testing/continuous/UndefinedAnalyzer.java]: May be able to use `Files.newBufferedReader()` here.  It is a new method in Java 1.8

File: src/main/java/org/apache/accumulo/testing/merkle/cli/GenerateHashes.java [hunk 2]
```diff
@@ -151,82 +129,78 @@ public void setSplitsFile(String splitsFile) {
...
     if (!client.tableOperations().exists(outputTableName)) {
       throw new IllegalArgumentException(outputTableName + " does not exist, please create it");
     }
 
     // Get some parallelism
     ExecutorService svc = Executors.newFixedThreadPool(numThreads);
-    final BatchWriter bw = client.createBatchWriter(outputTableName, new BatchWriterConfig());
 
-    try {
+    try (BatchWriter bw = client.createBatchWriter(outputTableName, new BatchWriterConfig())) {
```
Comment (keith-turner) [inline src/main/java/org/apache/accumulo/testing/merkle/cli/GenerateHashes.java]: Could this be `try (BatchWriter bw = client.createBatchWriter(outputTableName)) {`?

File: src/main/java/org/apache/accumulo/testing/merkle/cli/ManualComparison.java [hunk 3]
```diff
@@ -46,50 +46,51 @@ public static void main(String[] args) throws Exception {
     ManualComparisonOpts opts = new ManualComparisonOpts();
     opts.parseArgs("ManualComparison", args);
 
-    AccumuloClient client = opts.getClient();
+    try (AccumuloClient client = opts.createClient()) {
 
-    Scanner s1 = client.createScanner(opts.table1, Authorizations.EMPTY), s2 = client.createScanner(opts.table2, Authorizations.EMPTY);
-    Iterator<Entry<Key,Value>> iter1 = s1.iterator(), iter2 = s2.iterator();
-    boolean incrementFirst = true, incrementSecond = true;
+      Scanner s1 = client.createScanner(opts.table1, Authorizations.EMPTY), s2 = client.createScanner(opts.table2, Authorizations.EMPTY);
```
Comment (keith-turner) [inline src/main/java/org/apache/accumulo/testing/merkle/cli/ManualComparison.java]: Could possibly move creating these scanners into the try with resources block.

File: src/main/java/org/apache/accumulo/testing/stress/Scan.java [hunk 4]
```diff
@@ -38,28 +38,29 @@ public static void main(String[] args) throws Exception {
     ScanOpts opts = new ScanOpts();
     opts.parseArgs(Scan.class.getName(), args);
 
-    AccumuloClient client = opts.getClient();
-    Scanner scanner = client.createScanner(opts.getTableName(), new Authorizations());
+    try (AccumuloClient client = opts.createClient()) {
+      Scanner scanner = client.createScanner(opts.getTableName(), new Authorizations());
```
Comment (keith-turner) [inline src/main/java/org/apache/accumulo/testing/stress/Scan.java]: This scanner creation could be move into try

**Pull request merged**